### ISSUE TITLE
NTR: Phosphohydroxylysine high in urine (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8862]
New term request Phosphohydroxylysine high in urine (elthat_130413192044):

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000806
- label: Elevated urinary phosphohydroxylysine level
- parents: [Increased urinary non-proteinogenic amino acid level (HP:0033098)](https://hpo.jax.org/app/browse/term/HP:0033098)
- pmid: PMID:23242558
- def: The amount of phosphohydroxylysine in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8862
- synonyms:
